src/sentry/api/endpoints/event_apple_crash_report.py

```diff
@@ -30,9 +30,8 @@ def get(self, request, event_id):
         and not the event ID that is reported by the client upon submission.
         This works only if the event.platform == cocoa
         """
-        try:
-            event = Event.objects.get(id=event_id)
-        except Event.DoesNotExist:
+        event = Event.get_event(event_id)
+        if event is None:
```

Comment:
Damn, need that Python 3.8 `if not event := Event.get_event(event_id)` action up in here.